src/test/java/com/wikia/webdriver/testcases/widgettests/GoogleFormTests.java

```diff
@@ -38,7 +38,16 @@ public void GoogleFormWidgetTest_001_isLoaded() {
     widget.create(GOOGLE_FORM_ONE_WIDGET_ARTICLE_NAME);
     navigate.toPage(GOOGLE_FORM_ONE_WIDGET_ARTICLE_NAME);
 
-    Assertion.assertTrue(widget.isLoaded(), MercuryMessages.INVISIBLE_MSG);
+//    Assertion.assertTrue(widget.isLoaded(), MercuryMessages.INVISIBLE_MSG);
+
+    //Loaded set to false
+    Assertion.assertEquals("true", String.valueOf(widget.isLoaded()));
+
+//    Assertion.assertTrue(widget.isLoaded());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)